app-emulation/qemu/qemu-2.11.0.recipe

```diff
@@ -105,16 +109,25 @@ BUILD_PREREQUIRES="
 	cmd:ld$secondaryArchSuffix
 	cmd:make
 	cmd:find
+	cmd:iasl$secondaryArchSuffix
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:pod2man
 	cmd:python
 	"
 
+defineDebugInfoPackage qemu$secondaryArchSuffix \
+	"$binDir"/qemu-system-x86_64 \
+	"$binDir"/qemu-system-i386
+
 BUILD()
 {
-	export CFLAGS="-lbsd -lnetwork"
+	export CFLAGS="-lbsd -lnetwork \
+		-I`finddir B_SYSTEM_HEADERS_DIRECTORY`/capstone"
 	runConfigure --omit-buildspec ./configure \
-		--disable-stack-protector \
+		--iasl=`finddir B_SYSTEM_BIN_DIRECTORY`/iasl \
```

Comment:
The configure script already have [a default value for iasl][0]. As long as `iasl` could be found in `PATH`, it should work.

```diff
-		--iasl=`finddir B_SYSTEM_BIN_DIRECTORY`/iasl \
```

[0]: https://github.com/qemu/qemu/blob/v2.11.0/configure#L251